packages/wdio-browserstack-service/src/reporter.ts

```diff
@@ -67,16 +104,30 @@ export default class TestReporter extends WDIOReporter {
                 browser_version: this._capabilities?.browserVersion,
                 platform: this._capabilities?.platformName,
             }
+        }
 
-            const uploadData = {
-                event_type: 'TestRunFinished',
-                test_run: testData
-            }
+        if (eventType == 'TestRunSkipped') {
+            eventType = 'TestRunFinished'
```

Comment:
Why are we overriding this?

Reply:
Have used `TestRunSkipped` just to handle some conditions or things to do when tests are skipped. But ultimately, this needs to be send to backend as `TestRunFinished`. So overriding the `TestRunSkipped`event back to `TestRunFinished`

Reply:
So for those tests also where are called from `onTestSkip`, we were previously also sending it as a `TestRunFinished` event right? 
https://github.com/webdriverio/webdriverio/blob/a4e91659f0399fcd4ab71f1f2896229201cabb44/packages/wdio-browserstack-service/src/reporter.ts#L71

Reply:
So the test status is being sent as skipped and that reflects on reporting dashboard properly?

Reply:
Yes

Reply:
What do we send the event_type for skipped tests in other SDKs?